profile-service/src/main/resources/db/migration/common/V9.20__update_user_designated_body.sql

```diff
@@ -0,0 +1,13 @@
+UPDATE `UserDesignatedBody` set `designatedBodyCode` = '1-1RSG4X0' where `designatedBodyCode` = '1-AIIDQQ';
```

Comment:
Yorkshire and the Humber: "1-AIIDQQ"
North West: "1-AIIDNQ"
They are using the old codes on Stage because of the weekly synchronisation.